### PR TITLE
feat(tinacms): media manager dropzone accepts multiple files for upload

### PR DESCRIPTION
**Pull request by kendallstrautman**
Addresses some concerns from #1589

### CODE REVIEW

File: packages/tinacms/src/components/media/breadcrumb.tsx [hunk 1]
```diff
@@ -41,6 +41,11 @@ export function Breadcrumb({ directory = '', setDirectory }: BreadcrumbProps) {
       {directory &&
         directory.split('/').map((part, index, parts) => (
           <button
+            /** since these can't be reordered, using
+             * index as a part of key in case of dirs
+             * with the same name
+             * */
+            key={`${part}-${index}`}
```
Comment (kendallstrautman) [inline packages/tinacms/src/components/media/breadcrumb.tsx:48]: Let me know if there's a better way to handle this this key.

**Review by DirtyF — APPROVED** [review]
🙆‍♂️  I can drag and drop multiple PNG files in the media manager, they're showing fine when uploaded in the public folder.

**Comment by kendallstrautman** [comment]
Reminder to pair on the key issue @dwalkr

**Comment by DirtyF** [comment]
When you upload many images at the root folder, thumbnails are not shown

Demo: https://www.loom.com/share/195acac33d484175b74ec59d05523a8f

**Comment by dwalkr** [comment]
> When you upload many images in the default root folder, thumbnails are not shown, it seems to work only when they're in public (which is the root of the NextGitHubMediaStore)
> 
> Demo: https://www.loom.com/share/195acac33d484175b74ec59d05523a8f

I don't believe this issue to be related to this PR; it's due to how NextGitMedaStore is implemented (which is not a published package) and is, essentially, expected behavior in demo-next

**Review by dwalkr — APPROVED** [review]
👏👏👏👏

**Pull request merged**